routemaster/services/deliver.rb

```diff
@@ -40,6 +43,8 @@ def run
           response = _conn.post do |post|
             post.headers['Content-Type'] = 'application/json'
             post.body = data.to_json
+            post.options.timeout = TIMEOUT
+            post.options.open_timeout = CONNECT_TIMEOUT
```

Comment:
Any reason to make these part of the query code, as opposed to configuring them when configuring the Faraday connection (which gets reused?)